### PR TITLE
JNG-3902 Fix timestamp asstring

### PR DESCRIPTION
**Pull request by bencegelei**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-3902" title="JNG-3902" target="_blank">BugJNG-3902</a>  Timestamp.asString() produces wrong output
  </td></table>
  <br />
 

JNG-3902_Fix_timestamp_asstring

### CODE REVIEW

**Review by robertcsakany — APPROVED** [review]

File: docs/pages/04_types.adoc [hunk 1]
```diff
@@ -484,26 +484,31 @@ Time literals can be represented as literals using the following syntax.
...
 where
 
 * <hh> refers to a zero-padded hour between 00 and 23,
 * <mm> refers to a zero-padded minute between 00 and 59,
 * <ss> refers to a zero-padded second between 00 and 59,
+* <ff> refers to fractional second (millisecond) between 000 and 999 (trailing zeros can be omitted: `12:23:56.100 = 12:23:56.1`),
 * the surrounding backticks are required.
 
-The valid values of time are between `pass:[`00:00:00`]` and `pass:[`23:59:59`]`.
+The valid values of time are between `pass:[`00:00:00.000`]` (or `pass:[`00:00`]`) and `pass:[`23:59:59.999`]`.
```
Comment (gprivi) [inline docs/pages/04_types.adoc:498]: why is this necessary?

**Review by gprivi — CHANGES_REQUESTED** [review]

**Review by gaborflorian — APPROVED** [review]

File: docs/pages/04_types.adoc (hunk 1 above)
Comment (bencegelei) [reply docs/pages/04_types.adoc:498]: I updated the documentation to align with what the backend can handle and JSL syntax allows. However some of these capabilities can be hidden. TBD tomorrow
